updateReadme.py

```diff
@@ -82,7 +82,7 @@ def main():
         with open(
             os.path.join(data[key]["location"], README_FILENAME), "wt", encoding="utf-8"
         ) as out:
-            for line in open(README_TEMPLATE):
+            for line in open(README_TEMPLATE, encoding="utf-8"):
```

Comment:
That's what happens when we never test under Windows! 

Would have done the same. Good catch @XhmikosR :clap: